### DISCUSSION TITLE
Is it possible to hide .desktop files from other Desktop Environments?

### DISCUSSION THREAD

**Discussion by alicealysia** [abenz1267/walker#136] [General]
I've been using Walker with Niri for a while now, but something that's been getting in the way has been that if I search for files, two applications will appear:

![image](https://github.com/user-attachments/assets/ecde7407-9ec8-4045-a91c-5f5ae71f3219)

One of these is Nemo, which I want to use, the other is Nautilus, which I don't want to use. Nautilus has the OnlyShowIn=GNOME value set in its .desktop file. Is it possible to get walker to respect this?

**Comment by abenz1267** [comment]
Walker now respects `OnlyShowIn` and `NotShowIn`.

**Reply by alicealysia** [reply]
Legend, thank you!!!
